node-client.js

```diff
@@ -0,0 +1,35 @@
+/*
+  Copyright © 2018 Andrew Powell
+
+  This Source Code Form is subject to the terms of the Mozilla Public
+  License, v. 2.0. If a copy of the MPL was not distributed with this
+  file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+  The above copyright notice and this permission notice shall be
+  included in all copies or substantial portions of this Source Code Form.
+*/
+
+/**
+ * @note This file exists merely as an easy reference for folks adding it to their configuration entries
+ */
+
+(() => {
+  /* eslint-disable global-require */
+  const { run } = require('./lib/node-client');
```

Comment:
Maybe we can just
```
const { run } = typeof window === undefined ? require('./lib/node-client') : require('./lib/client/client')
```

Reply:
that would probably be better and keep the package cleaner